### PR TITLE
PR-related fixes

### PR DESCRIPTION
**Pull request by lgmugnier**
## Description of the change

> Description here (why is it needed, what does it do)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Refactor (code change with no functionality change)
- [ ] Enhancement (minor change below the level of a feature)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Remove feature (removes a feature we don't support anymore)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related JIRA tickets

Relates to JIRA: CWC-XXX

## Relevant release note information

Release Notes:

## Potential Security Impacts

Does this PR have any security impact?
Does it fix a security issue?
Does it add attack surface?
Why do we think this change is safe?

### Development

- [ ] Does the code changed/added as part of this pull request have test coverage?
- [ ] Do all tests related to the changed code pass in development?
- [ ] Have you ensured that at least one other person has tested your code?
- [ ] Have you tested the code?
- [ ] Have you attached any pictures (if relevant)?

### CODE REVIEW

File: bctl-go-daemon/bctl/daemon/plugin/kube/kube.go [hunk 1]
```diff
@@ -275,3 +275,16 @@ func (k *KubeDaemonPlugin) updateActionsMap(newAction IKubeDaemonAction, id stri
 	k.actions[id] = newAction
 	k.mapLock.Unlock()
 }
+
+// func (k *KubeDaemonPlugin) deleteActionsMap(rid string) {
```
Comment (sidpremkumar) [inline bctl-go-daemon/bctl/daemon/plugin/kube/kube.go:279]: I think this needs to be uncommneted

Comment (lgmugnier) [reply bctl-go-daemon/bctl/daemon/plugin/kube/kube.go:279]: but we don't delete anything from the actions map ever so if you uncomment it you'll see that it's not used

**Review by sidpremkumar — APPROVED** [review]
except that 1 comment

File: bctl-go-daemon/bctl/daemon/plugin/kube/kube.go (hunk 1 above)
Comment (lgmugnier) [reply bctl-go-daemon/bctl/daemon/plugin/kube/kube.go:279]: I left it in because we need to address that

**Pull request merged**